libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -66,22 +66,23 @@ const static uint32_t   default_context_free_discount_net_usage_num  = 20; // TO
-const static uint32_t   default_max_transaction_cpu_usage           = 3*default_max_block_cpu_usage/4; /// max trx cpu usage in microseconds
-const static uint32_t   default_min_transaction_cpu_usage           = 100; /// min trx cpu usage in microseconds (10000 TPS equiv)
-const static uint32_t   default_subjective_cpu_leeway_us            = 31000; /// default subjective cpu leeway in microseconds
-
-const static uint32_t   default_max_trx_lifetime               = 60*60; // 1 hour
-const static uint32_t   default_deferred_trx_expiration_window = 10*60; // 10 minutes
-const static uint32_t   default_max_trx_delay                  = 45*24*3600; // 45 days
-const static uint32_t   default_max_inline_action_size         = 4 * 1024;   // 4 KB
-const static uint16_t   default_max_inline_action_depth        = 4;
-const static uint16_t   default_max_auth_depth                 = 6;
-const static uint32_t   default_sig_cpu_bill_pct               = 50 * percent_1; // billable percentage of signature recovery
-const static uint32_t   default_block_cpu_effort_pct           = 80 * percent_1; // percentage of block time used for producing block
-const static uint16_t   default_controller_thread_pool_size    = 2;
-const static uint32_t   default_max_variable_signature_length  = 16384u;
+const static uint32_t   default_max_block_cpu_usage                  = 200'000; /// max block cpu usage in microseconds
+const static uint32_t   default_target_block_cpu_usage_pct           = 10 * percent_1;
+const static uint32_t   default_max_transaction_cpu_usage            = 3*default_max_block_cpu_usage/4; /// max trx cpu usage in microseconds
+const static uint32_t   default_min_transaction_cpu_usage            = 100; /// min trx cpu usage in microseconds (10000 TPS equiv)
+const static uint32_t   default_subjective_cpu_leeway_us             = 31000; /// default subjective cpu leeway in microseconds
+
+const static uint32_t   default_max_trx_lifetime                     = 60*60; // 1 hour
+const static uint32_t   default_deferred_trx_expiration_window       = 10*60; // 10 minutes
+const static uint32_t   default_max_trx_delay                        = 45*24*3600; // 45 days
+const static uint32_t   default_max_inline_action_size               = 512 * 1024;   // 512 KB
+const static uint16_t   default_max_inline_action_depth              = 4;
+const static uint16_t   default_max_auth_depth                       = 6;
+const static uint32_t   default_sig_cpu_bill_pct                     = 50 * percent_1; // billable percentage of signature recovery
+const static uint32_t   default_block_cpu_effort_pct                 = 80 * percent_1; // percentage of block time used for producing block
+const static uint16_t   default_controller_thread_pool_size          = 2;
+const static uint32_t   default_max_variable_signature_length        = 16384u;
+const static uint32_t   default_max_nonprivileged_inline_action_size = 4 * 1024; // 4 KB
```

Comment:
I would prefer to not see these whitespace changes so it is easier to see in git when a value was changed.
This will change the default genesis. Are we okay with that?